### PR TITLE
Documentation and STAR prefix 

### PR DESCRIPTION
**Pull request by Hammarn**

### CODE REVIEW

**Comment by ewels** [comment]
Nice! I think you have two copies of the `Images` directory though..

**Comment by ewels** [comment]
Markdown files are also duplicated in `docs` and `RNABP2.0.md`

File: Docs/README.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+subread featureCounts - gene counts, biotype counts, rRNA estimation.
+##String Tie
+FPKMs for genes and transcripts
+## edgeR
+edgR is a bioconductor package for R and is used to do differntial extression analysis on RNAseq data.
+Rcreate MDS plot and sample pairwise distance heatmap / dendrogram
+[edgeR Bioconductor page](https://bioconductor.org/packages/release/bioc/html/edgeR.html)
+The output generated from edgeR process is a heatmap and a MDS-plot.
+The heatmap may look something like this:
+![Heatmap](images/heatmap.png =500x500))
```
Comment (ewels) [inline Docs/README.md]: Not sure if this syntax is specific to Mou? Not seen size specifications like this before. Breaks it on GitHub though, and probably most markdown processors..

**Comment by Hammarn** [comment]
ugh, I thought I fixed that.

**Comment by ewels** [comment]
Nice work!

**Pull request merged**